geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -473,6 +475,11 @@ public void close(BucketRegion bucketRegion) {
         }
       }
     }
+    if (isBucketRegion) {
```

Comment:
this code should be in a finally to make sure it happens

Reply:
ok.